ci-operator/step-registry/upi/conf/vsphere/upi-conf-vsphere-ref.yaml

```diff
@@ -14,6 +14,9 @@ ref:
     requests:
       cpu: 10m
       memory: 100Mi
+  dependencies:
+  - name: "release:latest"
+    env: OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE
```

Comment:
btw, if possible, also add one more print line in upi-conf-vsphere before running create-manifest.